wikidict/render.py

```diff
@@ -82,7 +82,7 @@ def find_section_definitions(
 
                 # Transform and clean the Wikicode
                 definition = process_templates(word, clean(code), locale)
-                if not definition:
+                if len(definition) < 2:
```

Comment:
Are you OK with that change @lasconic? Without it, we would end with "almost" empty definitions:
```shell
$ python -m wikidict sv --get-word Götet
Götet   

  1. (slang) Göteborg.
  2. :
```

Reply:
I don't have the time to test right now, but that could be a problem for french and the way we handle verbal forms.

Reply:
I updated the patch to prevent side effects on other locales.